eng/versioning.targets

```diff
@@ -43,6 +43,10 @@
 
     <!-- SDK sets product to assembly but we want it to be our product name -->
     <Product>Microsoft%AE .NET Core</Product>
+
+    <!-- Use the .NET Core product version for informational version description -->
+    <InformationalVersion>$(PlatformPackageVersion)</InformationalVersion>
```

Comment:
The `PlatformPackageVersion` is coincidentally tracking the product's branding version, I'd feel better if we inverted this.  Define `ProductVersion` property and then have `PlatformPackageVersion` and `InformationVersion` use that property.

Reply:
It's a little unfortunate that this cannot match the version that core-setup eventually uses when building the shared framework, since that's the thing that is customer visible.